misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobConsumer.kt

```diff
@@ -208,7 +211,12 @@ internal class SqsJobConsumer @Inject internal constructor(
                 )
                 Status.OK
               } catch (th: Throwable) {
-                log.error(th) { "error handling job from ${queue.queueName}" }
+                val mdcTags = TaggedLogger.getThreadLocalMdcContext()
+                  .also { TaggedLogger.resetThreadLocalMdcContext() }
+
+                // The underlying logger takes care of adding the mdc tags and cleaning them up when calling this way
```

Comment:
Could also push this documentation down to `wisp.logging.log` if others would find that helpful.

Reply:
Done. Even found an underlying extension function I can import and call from the same wisp file to make this an `log.error(th` instead of `log.log(Level.ERROR, th`.